bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -29,7 +29,7 @@
 			<parameter name="syncVolume" type="boolean">
 				<label>Sync Volume</label>
 				<description>Sync Volume across linked Music Cast models</description>
-				<default>true</default>
+				<default>false</default>
```

Comment:
This could be a small breaking change for some users, why is this default changed?

Reply:
I have changed this default as the README says that default is false "Sync volume across linked models (default=false)".
Furthermore, I consider false to be more practical as I don't want to automatically sync the volume by default.